CDCgov/cdc-open-viz
# Dev 8488

## Testing Steps

1. Run dashboard component
2. Add filter dropdown component and edit
3. Add a data filter
4. In the settings for the data filter, change the filter dropdown order and ensure the change is reflected in the preview dropdown
5. Ensure that "custom" option shows drag-n-drop options that reflect in the preview

## Self Review

- I have added testing steps for reviewers
- I have commented my code, particularly in hard-to-understand areas
- My changes generate no new warnings
- New and existing unit tests are passing

## Screenshots (if applicable)



## Additional Notes
